lesson1/zenameless1_hometask1.rb

```diff
@@ -0,0 +1,17 @@
+class Foo
+  @@counter = 0
+
+  def initialize
+    @@counter += 1
+    @instance_id = @@counter
+  end
+
+  def my_instance
+    @instance_id.even? ? :even : :odd
+  end
+
+end
+
+foos = Array.new(10) { Foo.new } 
+foos.sample.define_singleton_method('my_instance') { :uniq }
```

Comment:
Nice, we didn't learn `#define_method` yet, but good to know that you can manage it by yourself.